CMakeLists.txt

```diff
@@ -38,6 +38,18 @@ set(KWIVER_BINARY_DIR           "${CMAKE_BINARY_DIR}")
 set(KWIVER_CONFIG_FILE          "${KWIVER_BINARY_DIR}/kwiver-config.cmake")
 # Create initial setup script
 set(KWIVER_SETUP_SCRIPT_FILE    "${KWIVER_BINARY_DIR}/setup_KWIVER.sh" )
+if(WIN32)
```

Comment:
A .sh can actually be used on windows via a bash shell like cygwin or something, but I do not think a .bat can be used at all on linux.

I can take out the conditionals, it won't hurt anything on linux

if we move the ctest to use these env scripts we can ensure they are working